README.md

```diff
@@ -216,6 +216,7 @@ This file describe how the DUTs are connected and powered.
 ```
 lab-slave-XX:		The name of the slave (where XX is a number)
   dispatcher_ip: the IP where the slave could be contacted. In lava-docker it is the host IP since docker proxify TFTP from host to the slave.
+  have_cpuflag_kvm: Does the host running lab-slave-XX have KVM
```

Comment:
flag name is strange.  How about `host_has_kvm` ?

Reply:
You are right, the flag miss the fact it is host related. but I think we need to keep cpuflag since KVM could be misinterpreted with the software support(qemu).
So I propose host_has_cpuflag_kvm